apps/mcp/README.md

```
@@ -481,12 +609,220 @@
+
+   # Production Environment
+   curl -X GET https://mcp.gauzy.co/.well-known/oauth-protected-resource \
+     -H "Accept: application/json"
+
+   # Local Development
+   curl -X GET http://localhost:3001/.well-known/oauth-protected-resource \
+     -H "Accept: application/json"
+   ```
+
+2. **Generate Test JWT Token** (for development):
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)